test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the request of community members Ref: Pull Request 1074
```

Comment:
I would have thought it would be consistent with the rest of the system for it to be defined there.

While we're discussing these things, why on earth would anyone restrict to 80 columns in a GUI world? That is a throw back to when code had to be written for an 80x24 Wyse terminal. It makes the code harder to follow and you can fit less on the screen by doing it. Better to have the whole statement on a single line where at all practicable.